stdlib/SparseArrays/src/sparsematrix.jl

```diff
@@ -1543,8 +1533,7 @@ argument specifies a random number generator, see [Random Numbers](@ref).
 ```jldoctest; setup = :(using Random; Random.seed!(0))
 julia> sprandn(2, 2, 0.75)
 2×2 SparseMatrixCSC{Float64,Int64} with 2 stored entries:
-  [1, 2]  =  0.586617
-  [2, 2]  =  0.297336
+⠘
```

Comment:
For small matrices like this, the Braille spy plot is basically unreadable, and I think we should just switch over to using `print_matrix` with `⋅` for non-stored entries when the matrix is sufficiently small.  For example, define:
```jl
# this would be a nice function to have anyway?
function isstored(A::AbstractSparseMatrixCSC, i::Integer, j::Integer)
    rows = rowvals(A)
    for istored in nzrange(A, j) # could do binary search if the row indices are sorted?
        i == istored && return true
    end
    return false
end

Base.replace_in_print_matrix(A::SparseMatrixCSC,i::Integer,j::Integer,s::AbstractString) =
    isstored(A, i, j) ? s : Base.replace_with_centered_mark(s)
```
and then do e.g.
```jl
if size(A,1) ≤ 20 && size(A, 2) ≤ 20
    Base.print_matrix(io, A)
    return
end
```

Reply:
Thank you for that great draft!
I have included it into the code and I think it works great!